docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -153,7 +153,7 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
 
 ## Adaptive Batch Scheduler
 
-The Adaptive Batch Scheduler can automatically decide parallelisms of operators for batch jobs. If an operator is not set with a parallelism, the scheduler will decide parallelism for it according to the size of its consumed datasets (Note that the decided parallelism can only be a power of 2, see ["The decided parallelism can only be a power of 2"](#limitations-2) for details). This can bring many benefits:
+The Adaptive Batch Scheduler can automatically decide parallelisms of operators for batch jobs. If an operator is not set with a parallelism, the scheduler will decide parallelism for it according to the size of its consumed datasets (Note that the decided parallelism can only be a power of 2, see ["The decided parallelism will be a power of 2"](#limitations-2) for details). This can bring many benefits:
```

Comment:
How about to exclude this note? It is already in the limitation part, and I find it is complicating the understanding of the motivation part.